5_databases/week3/datamodal.sql

```diff
@@ -0,0 +1,79 @@
+-- Active: 1699540080106@@0.0.0.0@3306
+CREATE DATABASE mealsharing;
+Use mealsharing;
+
+CREATE TABLE `reservation` (
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`number_of_guests` INT NOT NULL,
+`meal_id` int(10) UNSIGNED,
+`created_date` DATE NOT NULL,
+`contact_phonenumber` VARCHAR(255) NOT NULL,
+`contact_name` VARCHAR(255) NOT NULL,
+`contact_email` VARCHAR(255) NULL,
+CONSTRAINT `fk_meal` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `meal` (
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`title` VARCHAR(255) NOT NULL,
+`description` TEXT,
+`location` VARCHAR(255) NOT NULL,
+`when` DATETIME NOT NULL,
+`max_reservations` int NOT NULL,
+`price` DECIMAL NOT NULL,
+`created_date` DATE NOT NULL)
+ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+CREATE TABLE `review` (
+`id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+`title` VARCHAR(255) NOT NULL,
+`description` TEXT,
+`meal_id` int(10) UNSIGNED,
+`stars` int NOT NULL,
+`created_date` DATE,
+CONSTRAINT `fk_meal_review` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
```

Comment:
Looks great. It is good to see that you thought about which columns should be NULL or not. 👍 

It is a good practice to define for each column if the value should be NULL or NOT NULL. If not defined it will be NULL. However, the next developer in your code would have to know that in order to understand the code.